src/utils/findParents.tsx

```diff
@@ -0,0 +1,10 @@
+import { PersonType } from '../types/PersonType';
+
+export const findParents = (people: PersonType[]) => {
+  return people.map(person => {
+    return Object.assign(person, {
+      mother: people.find(mother => mother.name === person.motherName) || null,
+      father: people.find(father => father.name === person.fatherName) || null,
+    });
+  });
```

Comment:
```suggestion
  return people.map(person => ({
    ...person,
    mother: people.find(mother => mother.name === person.motherName) || null,
    father: people.find(father => father.name === person.fatherName) || null,
  }));
```

Reply:
Hm, now it looks logical )
Tnx for review )